src/api/searchApi.ts

```diff
@@ -59,6 +61,25 @@ export const fetchTickets = async (params: FetchTicketsParams) => {
   return getTickets.data;
 };
 
+export const fetchCustomerTickets = async (params: FetchTicketsParams) => {
```

Comment:
Denne vil overta for det gamle søket etter hvert.
Da kan vi godt fjerne `Customer`-biten (det gjelder flere andre plasser også), men nå trengs den for at nytt og gammelt søk skal leve ved siden av hverandre